aws/src/main/java/aws/core/AwsSunstoneDeployer.java

```diff
@@ -23,11 +28,21 @@
  */
 public class AwsSunstoneDeployer extends AbstractSunstoneCloudDeployer {
     @Override
-    public void deploy(Annotation annotation, ExtensionContext ctx) {
+    public void deploy(Annotation annotation, ExtensionContext ctx) throws SunstoneException {
         verify(annotation);
-        WithAwsCfTemplate awsTemplateDefinition = (WithAwsCfTemplate) annotation;
-
         AwsSunstoneStore store = AwsSunstoneStore.get(ctx);
+
+        if (WithAwsCfTemplate.class.isAssignableFrom(annotation.annotationType())) {
+            deployCfTemplate((WithAwsCfTemplate) annotation, store);
+        } else if (WithAwsCfTemplateRepeatable.class.isAssignableFrom(annotation.annotationType())) {
+            for (WithAwsCfTemplate withAwsCfTemplate : ((WithAwsCfTemplateRepeatable) annotation).value()) {
+                deployCfTemplate(withAwsCfTemplate, store);
+            }
+        }
+    }
+
+    private void deployCfTemplate(WithAwsCfTemplate awsTemplateDefinition, AwsSunstoneStore store) throws
+        SunstoneException {
```

Comment:
`SunstoneException` can be removed.

Reply:
Oh, I didn't mean to throw unchecked exceptions. I have changed those to Sunstone exceptions. See the amended commit please.